src/scenes/gameOver.js

```diff
@@ -1,7 +1,20 @@
+let gameOverBtnImg;
+let gameOverBtn;
+
 const gameOver = {
     "setup": function()
     {
-
+		gameOverBtnImg = loadImage("assets/textures/ReturnButton.png",
+			() => {
+				gameOverBtnImg = scaleNearestNeighbor(gameOverBtnImg, 301, 103);
+				gameOverBtn = new ImageButton(gameOverBtnImg, 13, 359,
```

Comment:
The issue that I see here is going to be that some of the function signatures have changed for ImageButtons since the last time main has been merged into this branch. My suggestion would be to merge main back into this branch, fix the method signatures and then re-commit before merging this in with main. In general, the show function should be as simple as copy from the old function to the new function